Netkan/Sources/Curse/CurseApi.cs

```diff
@@ -8,10 +8,10 @@ namespace CKAN.NetKAN.Sources.Curse
 {
     internal sealed class CurseApi : ICurseApi
     {
-        private static readonly ILog Log = LogManager.GetLogger(typeof (CurseApi));
+        private static readonly ILog Log = LogManager.GetLogger(typeof(CurseApi));
 
-        public static string CurseApiBase = "https://widget.mcf.li/project/";
-        public static string CurseApiEnd = ".json";
+        private static string CurseApiBase = "https://api.cfwidget.com/project/";
+        private static string CurseApiEnd = "";
```

Comment:
If we don't use this anymore, we might as well remove it.

Reply:
OK, will do.